app/src/main/java/com/keylesspalace/tusky/entity/ProxyConfiguration.kt

```diff
@@ -0,0 +1,32 @@
+package com.keylesspalace.tusky.entity
+
+import java.net.IDN
+
+class ProxyConfiguration private constructor(
```

Comment:
The entity package is only for models returned by the Mastodon api, this should be placed somewhere else, maybe settings?